src/components/edit/block-dnd/DraggableChordSymbol.tsx

```diff
@@ -18,39 +17,54 @@ interface DraggableChordSymbolProps extends ChordSymbolProps {
     }) => void;
 }
 
-const DraggableChordSymbol: React.FC<DraggableChordSymbolProps> = (
-    props: DraggableChordSymbolProps
-): JSX.Element => {
-    const [, dragRef] = useDrag({
-        item: NewDNDChord(props.chordBlockID, props.children),
-        end: (chord: DNDChord | undefined, monitor: DragSourceMonitor) => {
-            if (chord === undefined) {
-                return;
-            }
-
-            const dropResult = monitor.getDropResult();
-
-            if (!isDropResult(dropResult)) {
-                return;
-            }
-
-            props.onDrop({
-                chord: chord.chord,
-                sourceBlockID: chord.sourceBlockID,
-                destinationBlockID: dropResult.blockID,
-                splitIndex: dropResult.tokenIndex,
-                dropType: dropResult.dropEffect,
-            });
-        },
-    });
-
-    return (
-        <RootRef rootRef={dragRef}>
-            <ChordSymbol className={props.className}>
+const DraggableChordSymbol = React.forwardRef(
+    (
+        props: DraggableChordSymbolProps,
+        ref: React.ForwardedRef<Element>
+    ): JSX.Element => {
+        const [, dragRef] = useDrag({
+            item: NewDNDChord(props.chordBlockID, props.children),
+            end: (chord: DNDChord | undefined, monitor: DragSourceMonitor) => {
+                if (chord === undefined) {
+                    return;
+                }
+
+                const dropResult = monitor.getDropResult();
+
+                if (!isDropResult(dropResult)) {
+                    return;
+                }
+
+                props.onDrop({
+                    chord: chord.chord,
+                    sourceBlockID: chord.sourceBlockID,
+                    destinationBlockID: dropResult.blockID,
+                    splitIndex: dropResult.tokenIndex,
+                    dropType: dropResult.dropEffect,
+                });
+            },
+        });
+
+        const combinedRefCallback = useCallback(
```

Comment:
a technique to combine multiple refs by using a callback ref that assigns to the other refs.

a refactoring opportunity in the future for generic refs but trying not to jump the gun